engine/src/main/scala/cromwell/webservice/routes/wes/WesRunRoutes.scala

```diff
@@ -20,42 +27,49 @@ trait WesRunRoutes {
 
   lazy val runRoutes: Route =
     pathPrefix("ga4gh" / "wes" / "v1") {
+      concat(
         pathPrefix("runs") {
-          pathEnd {
```

Comment:
Do you not still need the `pathEnd` to indicate "if there's nothing else in the URL, do this"?

Reply:
I'm curious as to how this slipped in here. On my current branch, this `pathEnd` is not present.